### PR TITLE
Added data source for local alpine repository

### PR DESCRIPTION
**Pull request by Turhsus**
Going to do these 1 at a time instead of all at once.

### CODE REVIEW

File: docs/data-sources/repository/local/alpine_repository.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+# Artifactory Local Alpine Repository Data Source
+
+Provides a data source for alpine repositories.
+
+## Example Usage
+
+```hcl
+data "artifactory_local_alpine_repository" "terraform-local-test-alpine-repo-basic" {
+  key                 = "terraform-local-test-alpine-repo-basic"
```
Comment (alexhung) [inline docs/data-sources/repository/local/alpine_repository.md:13]: Extraneous spaces between attribute and `=` character

File: pkg/artifactory/datasource/repository/local/datasource_artifactory_local_alpine_repository.go [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+				PackageType: "alpine",
+				Rclass:      rclass,
+			},
+		}, nil
+	}
+
+	return &schema.Resource{
+		Schema:      local.AlpineLocalSchema,
+		ReadContext: MkRepoReadDataSource(packer.Default(local.AlpineLocalSchema), constructor),
+		Description: "Dummy description",
```
Comment (alexhung) [inline pkg/artifactory/datasource/repository/local/datasource_artifactory_local_alpine_repository.go]: Please update this with final description text

File: pkg/artifactory/datasource/repository/local/datasource_artifactory_local_repository_test.go [hunk 3]
```diff
@@ -63,3 +64,90 @@ func mkTestCase(repoType string, t *testing.T) (*testing.T, resource.TestCase) {
...
+				]
+			}
+		}
+		resource "artifactory_local_alpine_repository" "{{ .repo_name }}" {
+			key 	     = "{{ .repo_name }}"
+			primary_keypair_ref = artifactory_keypair.{{ .kp_name }}.pair_name
+			depends_on = [artifactory_keypair.{{ .kp_name }}]
+		}
+
+    data "artifactory_local_alpine_repository" "{{ .repo_name }}" {
```
Comment (alexhung) [inline pkg/artifactory/datasource/repository/local/datasource_artifactory_local_repository_test.go:129]: Indentations seem to be inconsistent.

**Review by alexhung — DISMISSED** [review]
Look good in general. While I don't have problem with creating a number of small PRs to make them easier to review, we need to ensure these PRs are created/reviewed/merged within reasonable timeframe (a few days?). I'd like to have these local repos data source released as few minor version releases as possible.

File: pkg/artifactory/datasource/repository/local/datasource_artifactory_local_repository_test.go (hunk 3 above)
Comment (Turhsus) [reply pkg/artifactory/datasource/repository/local/datasource_artifactory_local_repository_test.go:129]: Damn this is an intellij settings thing: tabs vs spaces default or something like that, will fix tomorrow

**Comment by Turhsus** [comment]
> Look good in general. While I don't have problem with creating a number of small PRs to make them easier to review, we need to ensure these PRs are created/reviewed/merged within reasonable timeframe (a few days?). I'd like to have these local repos data source released as few minor version releases as possible.

Should I just update this PR with all the other data sources?

**Comment by Turhsus** [comment]
Closing this PR in lieu of one large PR for all remaining local data sources

**Pull request closed without merging**